src/main/java/org/cactoos/iterable/Matched.java

```diff
@@ -54,7 +54,9 @@
      * @param fst The first part of duplex iterator.
      * @param snd The second part of duplex iterator.
      */
-    public Matched(final Iterable<X> fst, final Iterable<X> snd) {
+    public Matched(
+        final Iterable<? extends X> fst, final Iterable<? extends X> snd
```

Comment:
@andreoss Please, break the line here, putting one parameter for line.

Reply:
@fabriciofx fixed